postgresql-pgdg.te

```diff
@@ -1,3 +1,17 @@
 policy_module(postgresql-pgdg, 1.1.0)
```

Comment:
I guess the version of the module should be updated to 1.2.0 to be consistent

Reply:
Bien vu !